### ISSUE TITLE
about retraining imagenet on detNAS

### ISSUE THREAD

**Issue by twmht** [open-mmlab/mmrazor#200]
Hi,

After searching the architecutre from detNAS, the algorithm retrain the architecture on the imagenet, but why not just finetune from the imagenet supernet?

I did not see the paper to disucss this. 

any idea?

**Comment by humu789** [comment]
In my opinion, in the supernet stage, each subnet is trained randomly just to search for the best subnet better. But the supernet is not trained enough and not optimized well, you can see it from the loss and the evaluation result. So fine-tuning from the supernet is not better than retraining.

**Comment by twmht** [comment]
I have finetuned from AutoSlim weight, the weight is well optimized and the accuracy is comparable with retraining. B
